model/src/guild/audit_log/event_type.rs

```diff
@@ -66,15 +66,15 @@ pub enum AuditLogEventType {
     /// [Member]: super::super::Member
     /// [role]: super::super::Role
     MemberRoleUpdate = 25,
-    /// [Member] was moved to a [voice channel].
+    /// [Member] was moved between audio [channel]s.
     ///
     /// [Member]: super::super::Member
-    /// [voice channel]: crate::channel::VoiceChannel
+    /// [channel]: crate::channel::Channel
     MemberMove = 26,
-    /// [Member] was disconnected from a [voice channel].
+    /// [Member] was disconnected from an audio [channel].
```

Comment:
Call it voice for consistency.

```suggestion
    /// [Member] was disconnected from a voice [channel].
```